src/main/java/fi/okm/jod/yksilo/entity/Osaaminen.java

```diff
@@ -45,6 +38,9 @@ public class Osaaminen {
   @BatchSize(size = 1000)
   private Map<Kieli, Kaannos> kaannos;
 
+  @ManyToMany(mappedBy = "osaamisKiinnostukset")
+  private Set<Yksilo> kiinnostuneet;
+
```

Comment:
This association must be unidirectional (Yksilo->Osaaminen), please remove this.